lib/imgix/client.rb

```diff
@@ -68,7 +69,7 @@ def purge(path)
       res
     end
 
-    def prefix(path)
+    def prefix(_path)
```

Comment:
Yessir! I believe it was rubocop-autocorrected, but yeah I think it prefixed it because it saw the parameter wasn't being used.

Reply:
Will that potentially throw an error if users are passing a value using named arguments?

Reply:
You need to have `_path:` for it to be a named argument I believe. Source: been burned too many times by named arguments in ruby/rails

Reply:
Yeah so in this case, would a user invoking `prefix(path: '')` have to switch to `prefix(_path: '')`?

Reply:
**TL;DR** Yes, if we `path` was declared to be a named argument, but we have not.

Had we declared the parameter like so:

```ruby
def prefix(path:)
end
```
then yes this would break code using `prefix(path:  "")` once we switched to `prefix(_path: "")` but we haven't declared `path` or `_path` to be a key-word parameter, so we're good.

The would-be culprit is `:` which is necessary to inform ruby we are declaring a key-word parameter, but without it we are just declaring a regular parameter.